src/app/dataset-view/additional-components/lineage-component/services/lineage-graph-builder.service.ts

```diff
@@ -108,15 +108,17 @@ export class LineageGraphBuilderService {
         const qualifyingDatasets = uniqueDatasets.filter(
             (dataset: DatasetLineageBasicsFragment) =>
                 dataset.kind === DatasetKind.Root &&
-                dataset.metadata.currentSource?.fetch.__typename === "FetchStepUrl",
+                dataset.metadata.currentPollingSource?.fetch.__typename === "FetchStepUrl",
```

Comment:
Interesting: are we correctly producing lineage graphs, if root datasets are based on push sources?

Reply:
Created new ticket